package.json

```diff
@@ -2,42 +2,66 @@
   "name": "@google-cloud/trace-agent",
   "version": "2.1.3",
   "description": "Node.js Support for StackDriver Trace",
-  "main": "index.js",
-  "repository": "GoogleCloudPlatform/cloud-trace-nodejs",
+  "main": "build/index.js",
```

Comment:
Eventually would this become `build/src/index.js`?

Reply:
Yep, in a future PR